packages/playwright-core/src/utils/traceUtils.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ClientSideCallMetadata } from '@protocol/channels';
+
+export function serializseClientSideCallMetadata(metadatas: ClientSideCallMetadata[]): Buffer {
```

Comment:
I wonder why serialize and parse functions are in different files and packages?

Reply:
Because we only serialize in core and only parse in web. There are no common modules between those.